torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1337,6 +1359,23 @@ def reference_sigmoid(x):
                                     dtypes=[torch.bfloat16]),
                        )
                        ),
+        UnaryUfuncInfo('lgamma',
+                       ref=reference_lgamma,
+                       decorators=(precisionOverride({torch.float16: 7e-1}),),
+                       dtypes=all_types_and(torch.bool),
+                       dtypesIfCPU=all_types_and(torch.bool, torch.bfloat16),
+                       dtypesIfCUDA=all_types_and(torch.bool, torch.half),
+                       skips=(
+                           SkipInfo('TestUnaryUfuncs', 'test_reference_numerics',
+                                    dtypes=[torch.bfloat16]),
```

Comment:
**Skipping Bfloat16**
```python
>>> import torch
>>> torch.tensor(500.0, dtype=torch.bfloat16)
tensor(500., dtype=torch.bfloat16)
>>> t = torch.tensor(500.0, dtype=torch.bfloat16)
>>> torch.lgamma(t)
tensor(2608., dtype=torch.bfloat16)
>>> torch.lgamma(t.to(torch.float32))
tensor(2605.1160)
```

Reply:
Good skip. This again highlights that we should break-up test_reference_numerics into different value ranges (like small values, large values, extremal values, for example).

Reply:
It would be helpful if `ranges` can be specified per dtype or for group of dtype, because we don't want to decrease the range of all dtypes just to be able to support one dtype with smaller range.